Cython/Compiler/ExprNodes.py

```diff
@@ -3340,14 +3340,16 @@ class FormattedValueNode(ExprNode):
     # {}-delimited portions of an f-string
     #
     # value           ExprNode                The expression itself
-    # conversion_char str or None             Type conversion (!s, !r, !a, or none, or 'd' for integer conversion)
+    # conversion_char str or None             Type conversion (!s, !r, !a, none, or 'd' for integer conversion)
     # format_spec     JoinedStrNode or None   Format string passed to __format__
+    # expr_text       str or NOne             If not None, the expression text
```

Comment:
a) "NOne" -> None
b) Maybe a clearer name (or at least a clearer comment) - `self_documenting_text`?

Reply:
Will fix that `None` and fix the comment :+1: